abl_tests/2d_hill_periodic/milestone.xml

```diff
@@ -0,0 +1,23 @@
+<ParameterList name="MueLu">
```

Comment:
Do you intend to keep the 2-D tests as part of the regression test suite? If yes, this should move to `reg_tests/test_cases`

Reply:
I deleted the 2D test cases